web-services/query/src/main/java/datawave/webservice/query/metric/QueryMetricsBean.java

```diff
@@ -140,6 +140,38 @@ public BaseQueryMetricListResponse query(@PathParam("id") @Required("id") String
         }
     }
 
+    /**
+     * Returns metrics for the current users queries that are identified by the id
+     *
+     * @param id
+     *
+     * @return datawave.webservice.result.QueryMetricListResponse
+     *
+     * @RequestHeader X-ProxiedEntitiesChain use when proxying request for user, by specifying a chain of DNs of the identities to proxy
+     * @RequestHeader X-ProxiedIssuersChain required when using X-ProxiedEntitiesChain, specify one issuer DN per subject DN listed in X-ProxiedEntitiesChain
+     * @HTTP 200 success
+     * @HTTP 500 internal server error
+     */
+    @GET
+    @POST
+    @Path("/id/{id}/subplans")
+    @Interceptors({RequiredInterceptor.class, ResponseInterceptor.class})
+    public BaseQueryMetricListResponse subplan(@PathParam("id") @Required("id") String id) {
+        if (queryMetricsWriterConfiguration.getUseRemoteService()) {
+            return remoteQueryMetricService.id(id);
```

Comment:
This remote call would return the query metrics page, not the subplans.  There needs to be a remoteQueryMetricService.subplan that delegates to the query metrics microservice.  The query metrics microservice would need to implement a subplan endpoint similar to how you have done it here.